ui/src/components/Drawer/index.vue

```diff
@@ -109,4 +111,8 @@ const drawerDefaultWidth = computed(() => {
 const closeDrawer = () => {
   emit('update:open', false);
 };
+
+const handleChangeTab = (tab: ContentType) => {
+  emit('update:content-type', tab);
+};
 </script>
```

Comment:
There does not seem to be any clear difference that requires significant analysis at this point. The code appears to work correctly according to the context provided. If you have more specific requirements or need more detailed reviews, please let me know so I can assist further.